File: Mixter/Domain/Identity/Events/UserConnected.cs
using System;

namespace Mixter.Domain.Identity.Events
{
    public struct UserConnected : IDomainEvent
    {
        public UserId UserId { get; private set; }

        public DateTime ConnectedAt { get; private set; }

        public UserConnected(UserId userId, DateTime connectedAt)
            : this()
        {
            UserId = userId;
            ConnectedAt = connectedAt;
        }

        public object GetAggregateId()
        {
            return UserId;
        }
    }
}

File: app/Domain/Identity/UserConnected.php
<?php

namespace App\Domain\Identity;

use App\Domain\IDomainEvent;

class UserConnected implements IDomainEvent {
    /**
     * @var UserId
     */
    private $userId;

    /**
     * @var SessionId
     */
    private $sessionId;

    /**
     * @var \DateTime
     */
    private $connectedAt;

    public function __construct(UserId $userId, SessionId $sessionId, \DateTime $connectedAt) {
        $this->userId = $userId;
        $this->sessionId = $sessionId;
        $this->connectedAt = $connectedAt;
    }

    /**
     * @return UserId
     */
    public function getUserId()
    {
        return $this->userId;
    }

    /**
     * @return SessionId
     */
    public function getSessionId()
    {
        return $this->sessionId;
    }

    /**
     * @return \DateTime
     */
    public function getConnectedAt()
    {
        return $this->connectedAt;
    }

    public function getAggregateId()
    {
        return $this->getSessionId()->getId();
    }
}